From: Qingqing <wangyiqing@kylinos.cn>
Date: Wed, 19 Aug 2026 09:25:10 +0000
Subject: Translated using Weblate (Malay)

Currently translated at 100.0% (49 of 49 strings)

Translation: openkylin-huanghe/touchscreen-settings
Translate-URL: http://weblate.openkylin.top/projects/openkylin-huanghe/touchscreen-settings/ms/
---
 src/touchscreen-settings/translations/ms.ts | 20 ++++++++++----------
 1 file changed, 10 insertions(+), 10 deletions(-)

diff --git a/src/touchscreen-settings/translations/ms.ts b/src/touchscreen-settings/translations/ms.ts
index 773cf91..14b8225 100644
--- a/src/touchscreen-settings/translations/ms.ts
+++ b/src/touchscreen-settings/translations/ms.ts
@@ -273,31 +273,31 @@
         <location filename="../calibration/touchscreen.cpp" line="28"/>
         <location filename="../calibration/touchscreen.cpp" line="75"/>
         <source>TouchCalibrate</source>
-        <translation type="unfinished"></translation>
+        <translation>PenentukuranSentuh</translation>
         <extra-contents_path>/TouchCalibrate/TouchCalibrate</extra-contents_path>
     </message>
     <message>
         <location filename="../calibration/touchscreen.cpp" line="63"/>
         <source>monitor</source>
-        <translation type="unfinished"></translation>
+        <translation>pemantau</translation>
         <extra-contents_path>/TouchCalibrate/monitor</extra-contents_path>
     </message>
     <message>
         <location filename="../calibration/touchscreen.cpp" line="66"/>
         <source>touch id</source>
-        <translation type="unfinished"></translation>
+        <translation>capaian id</translation>
         <extra-contents_path>/TouchCalibrate/touch id</extra-contents_path>
     </message>
     <message>
         <location filename="../calibration/touchscreen.cpp" line="69"/>
         <source>input device</source>
-        <translation type="unfinished"></translation>
+        <translation>peranti input</translation>
         <extra-contents_path>/TouchCalibrate/input device</extra-contents_path>
     </message>
     <message>
         <location filename="../calibration/touchscreen.cpp" line="72"/>
         <source>map</source>
-        <translation type="unfinished"></translation>
+        <translation>peta</translation>
         <extra-contents_path>/TouchCalibrate/map</extra-contents_path>
     </message>
 </context>
@@ -311,17 +311,17 @@
     <message>
         <location filename="../calibration/touchscreenwidget.cpp" line="168"/>
         <source>monitor</source>
-        <translation type="unfinished"></translation>
+        <translation>pemantau</translation>
     </message>
     <message>
         <location filename="../calibration/touchscreenwidget.cpp" line="192"/>
         <source>touch id</source>
-        <translation type="unfinished"></translation>
+        <translation>capaian id</translation>
     </message>
     <message>
         <location filename="../calibration/touchscreenwidget.cpp" line="216"/>
         <source>input device</source>
-        <translation type="unfinished"></translation>
+        <translation>peranti input</translation>
     </message>
     <message>
         <location filename="../calibration/touchscreenwidget.cpp" line="240"/>
@@ -336,7 +336,7 @@
     <message>
         <location filename="../calibration/touchscreenwidget.cpp" line="263"/>
         <source>map</source>
-        <translation type="unfinished"></translation>
+        <translation>peta</translation>
     </message>
     <message>
         <location filename="../calibration/touchscreenwidget.cpp" line="283"/>
@@ -346,7 +346,7 @@
     <message>
         <location filename="../calibration/touchscreenwidget.cpp" line="288"/>
         <source>calibration</source>
-        <translation type="unfinished"></translation>
+        <translation>penentukuran</translation>
     </message>
 </context>
 <context>
